src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/privatemessage/controllers/PrivateMessageController.java

```diff
@@ -168,14 +183,20 @@ PrivateMessageResponse update(@Valid @RequestBody final EditPrivateMessage editP
 
     final PrivateMessageView privateMessageView = lemmyPrivateMessageService.createPrivateMessageView(
         privateMessage);
-    return PrivateMessageResponse.builder().private_message_view(privateMessageView).build();
+    return PrivateMessageResponse.builder()
+        .private_message_view(privateMessageView)
+        .build();
   }
 
   @Operation(summary = "Delete a private message.")
-  @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "OK", content = {
-      @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = PrivateMessageResponse.class))}),
-      @ApiResponse(responseCode = "400", description = "Private Message Not Found", content = {
-          @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ApiError.class))})})
+  @ApiResponses(value = {@ApiResponse(responseCode = "200",
+      description = "OK",
+      content = {@Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
+          schema = @Schema(implementation = PrivateMessageResponse.class))}), @ApiResponse(
+      responseCode = "400",
```

Comment:
Response code for not found is 404.  400 means "Bad Request" indicating that the client did something wrong